### PR TITLE
[E2E] Fix monitor tests

### PR DESCRIPTION
**Pull request by elenaNaboko**

### CODE REVIEW

File: tests/e2e/tests/regression/monitor/monitor.e2e.ts [hunk 1]
```diff
@@ -52,9 +52,13 @@ test
         //Click on Stop Monitor button
         await t.click(monitorPage.runMonitorToggle);
         //Check for "Monitor is stopped." text
-        await t.expect(monitorPage.monitorIsStoppedText.innerText).eql('Profiler is stopped.');
-        //Check that no commands are displayed after "Monitor is stopped" text
-        await t.expect(monitorPage.monitorIsStoppedText.nextSibling().exists).notOk('No commands in monitor');
+        await t.expect(monitorPage.monitorIsStoppedText.innerText).eql('Profiler is paused.');
```
Comment (tanyatrayanava) [inline tests/e2e/tests/regression/monitor/monitor.e2e.ts:55]: add assertion

File: tests/e2e/tests/regression/monitor/monitor.e2e.ts [hunk 2]
```diff
@@ -52,9 +52,13 @@ test
...
+        //Get the last log line
+        const lastTimestamp = await monitorPage.monitorCommandLineTimestamp.nth(-1).textContent;
+        //Click on refresh keys to get new logs
+        await t.click(browserPage.refreshKeysButton);
+        //Check that no commands are displayed after monitor paused
+        await t.expect(monitorPage.monitorCommandLineTimestamp.nth(-1).textContent).eql(lastTimestamp);
```
Comment (tanyatrayanava) [inline tests/e2e/tests/regression/monitor/monitor.e2e.ts:61]: add assertion

**Review by tanyatrayanava — CHANGES_REQUESTED** [review]

File: tests/e2e/tests/regression/monitor/monitor.e2e.ts (hunk 1 above)
Comment (elenaNaboko) [reply tests/e2e/tests/regression/monitor/monitor.e2e.ts]: added

File: tests/e2e/tests/regression/monitor/monitor.e2e.ts (hunk 2 above)
Comment (elenaNaboko) [reply tests/e2e/tests/regression/monitor/monitor.e2e.ts]: added

**Review by tanyatrayanava — APPROVED** [review]

**Pull request merged**